WDAC-Policy-Wizard/app/src/Exceptions_Control.resx

```diff
@@ -117,7 +117,7 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <metadata name="column_Action.UserAddedColumn" type="System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
+  <metadata name="column_Action.UddedColumn" type="System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
```

Comment:
Is UddedColumn a typo?

Reply:
Good find; this caught my eye too. I looked into this last night and it was 'auto' changed but I never touched the Exceptions control.. so strange. I tested and it appears to not affect anything so I will leave it as is